.github/workflows/deploy_to_environment.yml

```diff
@@ -81,16 +62,24 @@ jobs:
         run: |
           ./scripts/deploy_service "easi-app-db-migrate" "$ENV-ecs-manager-docker" "$LAMBDA_VERSION" "easi-db-migrate" "$ECR_REGISTRY/$ECR_REPOSITORY:$GIT_HASH"
           ./scripts/db_lambda_invoke "$ENV-ecs-manager-docker" "$LAMBDA_VERSION" "easi-app-db-migrate"
```

Comment:
Should we cut this over to use `run_ecs_task.sh` like we do in MINT?

https://github.com/CMSgov/mint-app/blob/f4c19e1c2c2c6aa1b692eb2ca81f1a8205cb2acb/.github/workflows/deploy_to_environment.yml#L65-L75

Reply:
@ClayBenson94 Yes we will. I wanted to make sure that Easi app is working before updating this. But I will go ahead and update it now.